modules/assim.batch/R/pda.bayestools.helpers.R

```diff
@@ -0,0 +1,233 @@
+##' Create priors for BayesianTools
+##' 
+##' Helper function for creating log-priors compatible with BayesianTools package
+##'
+##' @param prior.sel `data.frame` containing prior distributions of the selected parameters
+##'
+##' @return out Prior class object for BayesianTools package
+##' @details `prior.sel` must contain the following columns:
+##'   * `distn` -- String describing a distribution; e.g. `norm` for `dnorm`, `rnorm`, etc.
+##'   * `parama`, `paramb` -- First and second parameters, respectively, of the corresponding distribution
+##'
+##' Optionally, `prior.sel` may also contain the following columns:
+##'   * `param_name` -- Parameter name, which will be carried through to the prior object and sampler
+##'   * `lower`, `upper` -- Lower and upper bounds, respectively. These can be leveraged by the BayesianTools samplers.
+##'   * `best` -- Best guess for a parameter estimate. BayesianTools can also use this, though I'm not sure how...
```

Comment:
I imagine they somehow use it for initial conditions, but I can't find any documentation on it in the `BayesTools` package.